src/env.cpp

```diff
@@ -245,6 +245,8 @@ static void setup_path() {
         std::string cspath;
         cspath.resize(confstr(_CS_PATH, nullptr, 0));
         confstr(_CS_PATH, &cspath[0], cspath.length());
+        // remove the trailing null-terminator
+        cspath.resize(cspath.length() - 1);
```

Comment:
This needs to be guarded against length == 0.

Reply:
Fixed, thanks 👍